### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sdkottegoda**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1476,11 +1477,11 @@
...
 
         <!-- Axis2 Version -->
-        <axis2.version>1.6.1-wso2v31</axis2.version>
-        <axis2.wso2.version>1.6.1-wso2v31</axis2.wso2.version>
+        <axis2.version>1.6.1-wso2v34</axis2.version>
+        <axis2.wso2.version>1.6.1-wso2v34</axis2.wso2.version>
         <exp.pkg.version.axis2>1.6.1</exp.pkg.version.axis2>
         <imp.pkg.version.axis2>[1.6.1, 1.7.0)</imp.pkg.version.axis2>
-        <axis2-transports.version>2.0.0-wso2v29</axis2-transports.version>
+        <axis2-transports.version>2.0.0.wso2v33</axis2-transports.version>
```
Comment (Nirothipan) [inline pom.xml]: 2.0.0-wso2v33 ?

File: pom.xml [hunk 2]
```diff
@@ -1433,35 +1433,36 @@
 
         <version.jettison>1.3.4</version.jettison>
 
-        <orbit.version.axis2>1.6.1-wso2v31</orbit.version.axis2>
+        <orbit.version.axis2>1.6.1.wso2v34</orbit.version.axis2>
```
Comment (Nirothipan) [inline pom.xml]: 1.6.1-wso2v34 	 ?

File: pom.xml (hunk 1 above)
Comment (sdkottegoda) [reply pom.xml]: updated. Thanks :)

File: pom.xml (hunk 2 above)
Comment (sdkottegoda) [reply pom.xml]: updated thanks :)

**Pull request merged**